resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Auditing Rules
+
+General Resource Profile Administration functions for manipulating a General Resource Profile's Auditing Rules. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_audit_rules()`
+
+```python
+def get_audit_rules(self, resource: str, class_name: str) -> Union[dict, bytes, None]:
+```
+
+#### 📄 Description
+
+Check the **Audit Rules** for a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** for which RACF should check the audit rules.
```

Comment:
```markdown
The **general resource profile** whose **Audit Rules** are being requested.
```

Reply:
Makes sense, this change has also been carried to applicable other `resource` parameters.